### PR TITLE
Black Frames

### PR DESCRIPTION
**Pull request by retokromer**
- better and more complete wording

### CODE REVIEW

File: _artifacts/black_frames.md [hunk 1]
```diff
@@ -8,9 +8,13 @@ tags:
 published: true
 ---
 
+We are considering black frames that have been generated during the digitisation process, not black frames that have been added in case of a bad film tear during printing (a method used for fixing an image negative, in order that it relains in sync with the sound negative).
```
Comment (ablwr) [inline _artifacts/black_frames.md]: Relains, remains?

Comment (ablwr) [inline _artifacts/black_frames.md]: I don't think we've traditionally used "we" in describing artifacts, preferring a *pedia-style entry, maybe would be better to phrase as "This is considering..." or "In the instance of..."

Comment (retokromer) [inline _artifacts/black_frames.md]: Indeed!

Comment (retokromer) [inline _artifacts/black_frames.md]: OK, I will try to change this, and the others I have prepared as well.

**Pull request merged**